src/openassetio-core/include/openassetio/hostAPI/Manager.hpp

```diff
@@ -145,6 +148,97 @@ class OPENASSETIO_CORE_EXPORT Manager {
    * @}
    */
 
+  /**
+   * @name Context Management
+   * @see @ref stable_resolution
+   *
+   * @{
+   */
+
+  /**
+   *  Creates a new Context for use with the manager.
+   *
+   *  @warning Contexts should never be directly constructed, always
+   *  use this method or @ref createChildContext to create a new one.
+   *
+   *  @see @ref createChildContext
+   *  @see @fqref{Context} "Context"
```

Comment:
[consider][minor] Here and below, missing `@return`.

Reply:
Do you think we need them in `C++` where the return type is already documented, and there is little ambiguity in the method itself? I could see merit if were talking about ownership etc... but as its a Ptr?

Reply:
Yeah that's fine, the rendered docs look OK to me, just wasn't sure re. consistency, hence the `[minor]`.